server/src/test/scala/controllers/application/EgateBanksControllerSpec.scala

```diff
@@ -0,0 +1,38 @@
+package controllers.application
+
+import module.DRTModule
+import org.scalatestplus.play.PlaySpec
+import play.api.test.Helpers._
+import play.api.test.{FakeRequest, Helpers}
+
+class EgateBanksControllerSpec extends PlaySpec {
+
+  "EgateBanksController" should {
+
+    "get port e-gate details updates" in {
+
+      val module = new DRTModule() {
+        override val isTestEnvironment: Boolean = true
+      }
+
+      val drtSystemInterface = module.provideDrtSystemInterface
+
+      val controller = new EgateBanksController(Helpers.stubControllerComponents(), drtSystemInterface)
+
+      val result = controller.getEgateBanksUpdates.apply(FakeRequest().withHeaders("X-Auth-Email" -> "test@test.com",
```

Comment:
Maybe put this first header value on a new line so it appears next to the others to improve readability